### PR TITLE
yum: fix yum version comparison

### PR DESCRIPTION
**Pull request by mkrizek**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
yum

##### ANSIBLE VERSION

```
devel
```

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: lib/ansible/modules/packaging/os/yum.py [hunk 1]
```diff
@@ -347,7 +348,7 @@ def is_group_env_installed(name, conf_file, installroot='/'):
     name_lower = name.lower()
 
     my = yum_base(conf_file, installroot)
-    if yum.__version__ >= '3.4':
+    if LooseVersion(yum.__version__) >= LooseVersion('3.4'):
```
Comment (Akasurde) [inline lib/ansible/modules/packaging/os/yum.py]: LooseVersion is broken in Python 3

Comment (mkrizek) [reply lib/ansible/modules/packaging/os/yum.py]: @abadger any hints on what is proper way of doing this?

Comment (abadger) [reply lib/ansible/modules/packaging/os/yum.py]: For yum, use ```yum.__version_info__``` to compare against.

**Review by Akasurde — APPROVED** [review]
LGTM

**Pull request merged**